### PR TITLE
Set default session duration

### PR DESCRIPTION
**Pull request by k0da**
Signed-off-by: Dinar Valeev <dinar.valeev@absa.africa>

### CODE REVIEW

**Review by ytsarev — COMMENTED** [review]
Could you please briefly describe the motivation behind this change?

File: chart/samlet/values.yaml [hunk 1]
```diff
@@ -16,5 +16,4 @@ samlet:
   #idpEndpoint: "https://example.com"
   idpEndpoint: ""
   # default validity time for issued secrets
-  #sessionDurattion: 1h
-  sessionDuration: ""
+  sessionDuration: "1h"
```
Comment (donovanmuller) [inline chart/samlet/values.yaml:19]: One value should change to be consistent with the other perhaps? See https://github.com/AbsaOSS/samlet/blob/4e100d002a47fb14af20194f647f3540b60f53f1/README.md#L35

Either the README comment should be updated or the default be updated to

```suggestion
  sessionDuration: "2h"
```

**Review by donovanmuller — CHANGES_REQUESTED** [review]

**Comment by k0da** [comment]
without default value being set it looks ugly
```yaml
    env:
    - name: IDP_ENDPOINT
      value: https://idp.example.com
    - name: SESSION_DURATION
```

File: chart/samlet/values.yaml (hunk 1 above)
Comment (k0da) [reply chart/samlet/values.yaml:19]: @donovanmuller those are two different things. Controller option sets 'global' default. While .spec.sessionDuration can override this per object.

**Review by donovanmuller — APPROVED** [review]

File: chart/samlet/values.yaml (hunk 1 above)
Comment (donovanmuller) [reply chart/samlet/values.yaml:19]: Understood, thanks for the clarification 👍

**Review by somaritane — APPROVED** [review]

**Pull request merged**